README.md

```diff
@@ -1,14 +1,44 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# My Portfolio
+# Portfolio.
 
-> My Microverse portfolio Desktop Version Final
+> In this branch (animation_transition) I used the Animation and transition to build attractive web design I did it with my collaborators and you can check the commits of my collaborators at this branch.
 
 ## Built With
 
-- HTML/CSS
+- HTML
+- CSS
+- Flexbox
 - Figma
--
+- Formspree service
+- (RWD) responsive web design using media query
+- Animation and Transition
+
+
+
+## Live Demo (if available)
```

Comment:
Kindly, I suggest you put the link to your page displayed here.

Reply:
> Kindly, I suggest you put the link to your page displayed here.

@mgmediaweb  could you please tell me a little clear in which line I should put and which link

Reply:
> Kindly, I suggest you put the link to your page displayed here.

@mgmediaweb you mentioned that my latest deployed version produces errors kindly please make it a little clear what kind of errors it has and could you please share a screenshot of the errors it is not obvious to me thanks a lot

Reply:
Line 19, you need remove "if available" and bellow put the URL to your deployed page

Reply:
@mgmediaweb  kindly could you please tell me more about the error which pordueced